docs/src/main/sphinx/connector/hive.md

```diff
@@ -800,8 +800,8 @@ WITH (format='CSV',
   -
 * - `orc_bloom_filter_columns`
   - Comma separated list of columns to use for ORC bloom filter. It improves the
-    performance of queries using range predicates when reading ORC files.
-    Requires ORC format.
+    performance of queries using equality predicates, such as `=` and `IN`, when
```

Comment:
See `io.trino.spi.predicate.ValueSet#tryExpandRanges`, small range predicates can be converted into discrete values for bloom filtering.
This should say something like `using equality, IN and small range predicates`